notifications/models/govuk-notify-model.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable consistent-return */
+const config = require('../../config');
+const logger = require('../../lib/logger')({ env: config.env });
+const {
+  caseworkerEmail,
+  successTemplateId,
+  failureTemplateId
+} = config.notifications;
+
+module.exports = class NotifyModel {
+  async sendCaseworkerNotification(client, jobReport) {
+    const {
+      success,
+      recordsCount,
+      errorMessage,
+      fileUploadTime,
+      jobEndedTime,
+      invalidRecords
+    } = jobReport;
+
+    const emailProps = {
+      csv_uploaded_datetime: this.getDateAndTimeString(fileUploadTime),
+      cepr_update_datetime: this.getDateAndTimeString(jobEndedTime),
+      has_invalid_records: (invalidRecords.length ? 'yes' : 'no'),
+      link_to_file: ''
```

Comment:
You have to set `link_to_file` to empty string? Can you just not set it or is it to help with readability and clarity for setting up the emailProps?

Reply:
This is because of a quirk of Notify where you can have conditional text based on a personalisation variable but not include another the variable itself within conditional text. 

If set as an empty string by default it will display nothing in the template, and if it is replaced with an actual link it will display this instead. It does help with clarity but the main reason is to avoid creating two new templates for situations where there is a link to display.